src/Services/busTrackerAPI.ts

```diff
@@ -1,6 +1,15 @@
 import axios from "axios";
 
 const BusTrackerAPI = {
+    getAllRoutes: async (format: string) => {
+        const response = await axios.get("https://cors-anywhere.herokuapp.com/http://www.ctabustracker.com/bustime/api/v2/getroutes", {
```

Comment:
`cors-anywhere.herokuapp.com`?

Reply:
That's there because I was running into cors/corb issues when trying to query the api:

`Access to XMLHttpRequest at 'http://www.ctabustracker.com/bustime/api/v2/getroutes?key=<key>&format=json' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.`

and 

`Cross-Origin Read Blocking (CORB) blocked cross-origin response http://www.ctabustracker.com/bustime/api/v2/getroutes?key=<key>&format=json with MIME type application/json. See https://www.chromestatus.com/feature/5629709824032768 for more details.`

I thought I could add `"proxy": "http://localhost:3000"` to my `package.json` to get around it but it didn't work.